editor/src/components/canvas/canvas-strategies/absolute-move-strategy.spec.tsx

```diff
@@ -217,7 +217,7 @@ describe('Absolute Move Strategy', () => {
       makeTestProjectCodeWithSnippet(
         `<View style={{ ...(props.style || {}) }} data-uid='aaa'>
         <View
-          style={{ backgroundColor: '#0091FFAA', position: 'absolute', top: 65, width: 250, height: 300, left: 15 }}
+          style={{ backgroundColor: '#0091FFAA', position: 'absolute', top: 65, width: 250, height: 300, left: 65 }}
```

Comment:
could you make a karma test for this? the fake metadata is making this test a bit too fake!

also, in the karma test, could you please add margin to the parent? that should shift the static element and we should respect that

Reply:
This is a good idea. It seems like we could add a few tests that shift the absolute positioned element to make sure we're not missing any other edge cases like this